myhpi/core/templates/core/sidebar.html

```diff
@@ -0,0 +1,25 @@
+{% load core_extras %}
+{% load i18n %}
+
+{% if perms.wagtail.edit_page %}
```

Comment:
This does not work for me. Do you have documentation mentioning this?

Reply:
Sadly, no, but it works on my side(tm).